packages/fxa-admin-server/src/scripts/audit-tokens.ts

```diff
@@ -9,6 +9,7 @@ import packageJson from '../../package.json';
 import Config from '../config';
 import mozlog from 'mozlog';
 import { ILogger } from 'fxa-shared/log';
+import { clean } from 'semver';
```

Comment:
Q: What's this?
I see we're importing it, but not sure where below we're using the new `clean()` method.

Reply:
Oh good catch. I am guessing VSCODE auto-completed this and I didn't catch it.